### ISSUE TITLE
Add support for application config -> matchspecification -> entity?

### ISSUE THREAD

**Issue by karamelba** [gessnerfl/terraform-provider-instana#84]
Hi, i'd like to understand how to set the entity ('Destination' or 'Source') within the  matchspecification of an application configuration.
The Instana APi https://instana.github.io/openapi/#operation/addApplicationConfig marks the field 'entity' as required. 
If i create a match specification like 'kubernetes.namespace EQUALS 'xyz' via the terraform-provider-instana, the result is an application config with a match specification of type 'Destination'.
How can i achieve a match specification  of type 'Source'?
Thanks a lot!

**Comment by gessnerfl** [comment]
@karamelba: this is currently not supported. I marked the ticket as a feature and will check how this can be supported

**Comment by karamelba** [comment]
Great! Thank you :-)

**Comment by gessnerfl** [comment]
I will adjust the match expression syntax such as that you can optionally specify the entity origin (source or destination) after the entity path. For compatibility it will be defaulted to destination when the origin is omitted.

Examples:
```bash
entity.name      # no entity origin specified -> defaults to destination
entity.name@src  # entity name at the source of the call
entity.name@dest # entity name at the destination of the call
```

**Comment by karamelba** [comment]
Sounds great! :-)

**Comment by gessnerfl** [comment]
Defaulting to destination is not a valid approach. There are entities which do not support source or destination. Will check if the API is defaulting the values properly. In the API entity is marked as required but this does not implemented like this.

**Comment by gessnerfl** [comment]
Feature is available with version 0.10.0

**Comment by karamelba** [comment]
Thanks a lot for this enhancement!
